cpp/src/cylon/join/join_config.hpp

```diff
@@ -184,12 +184,30 @@ class JoinConfig {
     return left_column_idx.size() > 1 || right_column_idx.size() > 1;
   }
 
- private:
+  bool is_use_ops() const {
```

Comment:
Aren't we using camel case for private functions?

Reply:
These are removed